package.json

```diff
@@ -75,7 +75,7 @@
     "spin.js": "^3.1.0",
     "styled-components": "^4.1.3",
     "superagent": "^5.1.0",
-    "webpack": "^4.42.0",
+    "webpack": "5.0.0-beta.16",
```

Comment:
pinned to this version rather than latest, because latest introduces a breaking change, which hasn't yet been picked by the html-webpack-plugin, making it impossible to buil